pkg/controller/elasticsearch/services/services.go

```diff
@@ -103,6 +103,7 @@ func NewExternalService(es esv1.Elasticsearch) *corev1.Service {
 }
 
 // IsServiceReady checks if a service has one or more ready endpoints.
+// *Note* this is unused now.  Should we simply remove? --mmontgomery
```

Comment:
Yes, I would remove this function as it is now dead code. Git is our kind memory to help us if we ever want to restore it.